client/dashboard/app/queries.ts

```diff
@@ -16,21 +16,21 @@ import {
 	updateSiteSettings,
 	restoreSitePlanSoftware,
 } from '../data';
-import { SITE_FIELDS } from '../data/constants';
+import { SITE_FIELDS, SITE_OPTIONS } from '../data/constants';
 import { queryClient } from './query-client';
 import type { Profile, SiteSettings, UrlPerformanceInsights } from '../data/types';
 import type { Query } from '@tanstack/react-query';
 
 export function sitesQuery() {
 	return {
-		queryKey: [ 'sites', SITE_FIELDS ],
+		queryKey: [ 'sites', SITE_FIELDS, SITE_OPTIONS ],
```

Comment:
I don't like that the query key keeps getting more complex. I'd rather have a random int in the key tbh.